app/controllers/find/track_controller.rb

```diff
@@ -6,9 +6,34 @@ def track_click
       utm_content = params[:utm_content]
       url = params[:url]
 
-      Analytics::ClickEvent.new(utm_content:, url:, request:).send_event
+      track_click_event(utm_content, url)
 
       redirect_to url, allow_other_host: true
     end
+
+    def track_apply_to_course_click
+      utm_content = params[:utm_content]
+      url = params[:url]
+
+      track_click_event(utm_content, url)
+
+      send_candidate_applies_analytics_event(params[:course], params[:candidate])
```

Comment:
I think is best to use current_candidate if logged in, no? 

Otherwise this parameter can be override by any user.